components/script/dom/shadowroot.rs

```diff
@@ -421,3 +421,12 @@ impl<'dom> LayoutShadowRootHelpers<'dom> for LayoutDom<'dom, ShadowRoot> {
         }
     }
 }
+
+impl From<ShadowRootMode> for devtools_traits::ShadowRootMode {
```

Comment:
Can you use [Convert](https://doc.servo.org/script/conversions/trait.Convert.html) instead of From here? That will mean one less change needed when splitting the script crate.